docs/sdk/sdk-by-examples/intro.md

```diff
@@ -0,0 +1,7 @@
+**SDK by Examples** offers an alternative and complementary way to learn about the Cosmos-SDK. It contains several examples that showcase how to build a decentralised application on top of the Cosmos-SDK from start to finish.  
+
+Without further due, let us get into it!
+
+- [Simple governance example](./simple-governance/intro.md)
```

Comment:
only one example for to start?

Reply:
yup, when we have other examples we'll add them to the list